src/modules/header/index.ts

```diff
@@ -1 +1,3 @@
-export { Header } from "./Header";
+import Header from "@modules/header/Header";
```

Comment:
If you are close enough yo the file, then use `./Header`.
This may help you to avoid issues with imports for example if you decide to add tests.